android/app/src/main/kotlin/com/linagora/android/tmail/MainActivity.kt

```diff
@@ -1,10 +1,23 @@
 package com.linagora.android.tmail
 
+import android.os.Bundle
+import android.os.PersistableBundle
 import io.flutter.embedding.android.FlutterFragmentActivity
 import io.flutter.embedding.engine.FlutterEngine
 import io.flutter.plugins.GeneratedPluginRegistrant
+import android.content.Intent.FLAG_ACTIVITY_NEW_TASK
 
 class MainActivity: FlutterFragmentActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
+        if (intent.getIntExtra("org.chromium.chrome.extra.TASK_ID", -1) == this.taskId) {
```

Comment:
what this use for?

Reply:
??

Reply:
https://pub.dev/packages/receive_sharing_intent 
follow document

Reply:
When I'm developing an app that receives data using intents(ACTION_SEND). I've discovered a weird issue when I'm sharing content from Chrome(only happens in Chrome 83 or above, older versions can't reproduce the issue) for instance, if the content is only selected text, for example, the data reaches my app correctly, but if the content shared is a URL, it reaches my app, but suddenly my app replaces Chrome in the app switcher. So, if I have my app and Chrome opened, after the data sharing I have two instances of my app(even if the Chrome icon appears in top of that window, if I tap it, opens my app).

This code to resolve this problem